src/test/java/com/fundacionjala/movies/RentalTest.java

```diff
@@ -0,0 +1,37 @@
+package com.fundacionjala.movies;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class RentalTest {
+
+    public static final String TITLE = "The Jungle Book";
+    public static final int DAYS_RENTED = 1;
+    private static final Movie MOVIE = new MovieChildren(TITLE);
```

Comment:
This is wrong
It is not a good practice to have instances in the attribute definition